src/rabbit_env.erl

```diff
@@ -1402,9 +1402,7 @@ load_conf_env_file(#{os_type := {unix, _},
                 true ->
                     case os:find_executable("sh") of
                         false -> Context1;
-                        Sh    -> do_load_conf_env_file(Context1,
-                                                       Sh,
-                                                       ConfEnvFile)
+                        Sh    -> do_load_conf_env_file(unix, Context1, Sh, ConfEnvFile)
```

Comment:
The OS type is available in `Context1`, you don't need to copy it to another argument.

Reply:
Done!